clang/test/SemaSYCL/esimd-private-statics.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify -pedantic %s
+// expected-no-diagnostics
+
+// This test checks that non-const statics are allowed for ESIMD
+
+static __attribute__((opencl_private)) __attribute__((sycl_explicit_simd)) int esimdPrivStatic;
+
+__attribute__((sycl_device)) __attribute__((sycl_explicit_simd)) void usage() {
+  // expected-note@+1{{called by}}
```

Comment:
I'm confused -- the test says `expected-no-diagnostics` but then it expects a note. What is generating the diagnostic the note is attached to? (Also, I would have guessed this `expected-note` would conflict with the `expected-no-diagnostics`, so that's... a bit of a surprise.)

Reply:
Oh, tee hee, I see the CI pipeline is now failing with the conflict between the expecteds, so huttah for my intuition being right. :-)

Reply:
Yeah, this is a leftover. It shouldn't be there. Deleted.